src/uniprotkb/components/results/utils.ts

```diff
@@ -0,0 +1,64 @@
+import queryStringModule from 'query-string';
```

Comment:
UniProtKB utils are now grouped at `./src/uniprotkb/utils/*`

Reply:
Moved. Although it feels weird to have a generic `utils` with utils files which are then specific to other locations...maybe we can improve this later.